src/raudio.c

```diff
@@ -2319,7 +2319,7 @@ static ma_uint32 ReadAudioBufferFramesInInternalFormat(AudioBuffer *audioBuffer,
     {
         // We break from this loop differently depending on the buffer's usage
         //  - For static buffers, we simply fill as much data as we can
-        //  - For streaming buffers we only fill the halves of the buffer that are processed
```

Comment:
raudio.c line 2322: I am unclear on what change captures the intended sense here.  I think it is "**a half of the buffer that is processed**."  That might mean both haves in sequence when the buffer has been completely drained.

Reply:
good